frontend/pages/category.js

```diff
@@ -43,10 +42,10 @@ class Category extends Component {
       );
     });
     return (
-      <Menu menu={this.props.headerMenu} />
-      <h1>{this.props.categories[0].name} Posts</h1>
       <div>
-        {postsMarkup}
+        <Menu menu={this.props.headerMenu} />
```

Comment:
Save yourself some typing and start destructuring these. 
```
const {headerMenu, categories} = this.props;
```

Reply:
where do I put that?